website/admin/views/admin_user_devices.py

```diff
@@ -0,0 +1,84 @@
+from flask import render_template, flash, redirect, url_for
+
+import data_interface
+from admin import admin_site
+from website.forms import AddNewDeviceForm, SetThermostatTargetForm
```

Comment:
There is no module named website. The root directory of the website component is already that directory. If you want to keep the forms.py file there, you should import it as:
from forms import ...